roles/builder/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set fact
+  when: "'-' in builder_compose_type"
+  ansible.builtin.set_fact:
+    a: "{{ builder_compose_type }}"
+    builder_compose_type: "{{ builder_compose_type.split('-')[0] + '-commit' if builder_compose_type.split('-')[1] == 'installer' else builder_compose_type }}"
```

Comment:
Suggestion: rename this variable to `_image_build_compose_type` and use that in the block below, and keep the original `builder_compose_type` variable as is